src/view/components/countries/SelectedCountries.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from "react";
+import { CountryName } from "../../../data/constants/CountryEmissions";
+import { TileTooltip } from "../atomic/TileTooltip";
+
+type SelectedCountriesType = {
+    selectedCountries: Map<CountryName, number>;
+    removeSelectedCountry: (country: CountryName) => void;
+    setCountryPercentage: (country: CountryName, percentage: number) => void;
+};
+
+export const SelectedCountries = ({
+    selectedCountries,
+    removeSelectedCountry,
+    setCountryPercentage,
+}: SelectedCountriesType) => {
+    const [worldPercentage, setWorldPercentage] = React.useState(1);
+    const [isPercentageError, setIsPercentageError] = React.useState(false);
+    const percentageError = "Percentages must stay between 0% and 100%";
+
+    useEffect(() => {
+        let totalPercentage = 0;
+        selectedCountries.forEach((percentage) => {
+            totalPercentage += percentage;
+        });
+        setWorldPercentage(1 - totalPercentage);
+    }, [selectedCountries]);
+
+    useEffect(() => {
+        let countryHasError = false;
+        selectedCountries.forEach((percentage) => {
+            if (percentage > 1 || percentage < 0) countryHasError = true;
+        });
```

Comment:
You can swap this out for `const hasPercentageError = selectedCountries.some((percentage) => percentage > 1 || percentage < 0)